src/models.py

```diff
@@ -0,0 +1,36 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+
+import auth_util
+
+db = SQLAlchemy()
+migrate = Migrate()
+
+class User(db.Model):
+    user_id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(120), unique=True)
+    hashed_pw = db.Column(db.String(120))
+    email_verified = db.Column(db.Boolean())
+    email_verification_secret = db.Column(db.String(64))
+
```

Comment:
Let's go ahead and also add a `created_at` column. Will need to `import datetime` at the top.
```
created_at = db.Column(db.DateTime(), default=datetime.datetime.utcnow)
```